doc/release-notes/SONiC_202405_Release_Notes.md

```diff
@@ -0,0 +1,109 @@
+| ***[NTP] Fix config template to init default parameters*** | This implements the fix for NTP config generation from the minigraph and save backward compatability | [18736](https://github.com/sonic-net/sonic-buildimage/pull/18736) | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***[SubnetDecap] Add subnet decap HLD*** | This feature implements the subnet decapsulation feature on T0 SONiC that allows Netscan to probe VLAN subnet IP addresses.  | [1657](https://github.com/sonic-net/SONiC/pull/1657)  | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***Add details of sff_mgr regarding deterministic bringup for SFF compliant modules*** | This feature adds a new thread sff_mgr under xcvrd to provide deterministic link bringup feature for SFF compliant modules (100G/40G) | [1334](https://github.com/sonic-net/SONiC/pull/1334) | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***Add HLD for IPv4 port based DHCP server in SONiC*** | This feature implements the design details of ipv4 port based DHCP server in SONiC.  | [1282](https://github.com/sonic-net/SONiC/pull/1282) |  [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***Add LDAP HLD*** | This ldap features describes the requirements, architecture and configuration details of ldap feature in switches for SONiC OS build image.  | [1487](https://github.com/sonic-net/SONiC/pull/1487) | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***Add SRv6 SID L3Adj*** | This feature describes the extensions of SRv6Orch required to support the programming of the L3Adj associated with SRv6 uA, End.X, uDX4, uDX6, End.DX4, and End.DX6 behaviors.     |  [1472](https://github.com/sonic-net/SONiC/pull/1472)     | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***Base OS upgrade to Bookworm*** | This feature upgrades SONiC's base image from Debian Bullseye to Debian Bookworm with the notable changes. | [17234](https://github.com/sonic-net/sonic-buildimage/pull/17234)  | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
+| ***Bookworm Upgrade LLDP, SNMP subagent, ICCPD, PDE*** | This feature implements the bookworm upgrade for LLDP, SNMP subagent, ICCPD, PDE |  [1677](https://github.com/sonic-net/SONiC/issues/1677)     | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |
```

Comment:
The set of code PR also includes FRR bookworm migration, pls modify to

 | ***Bookworm Upgrade LLDP, SNMP subagent, ICCPD, PDE, FRR*** | This feature implements the bookworm upgrade for LLDP, SNMP subagent, ICCPD, PDE, FRR | [1677](https://github.com/sonic-net/SONiC/issues/1677) | [Alpha](https://github.com/sonic-net/SONiC/blob/master/doc/SONiC%20feature%20quality%20definition.md) |

Reply:
Updated as suggested.